### PR TITLE
feat(relay): default portal URL

### PR DESCRIPTION
**Pull request by thomaseizinger**
Instead of having portal URL and token optional, we default the portal URL and decide based on the presence of the token, whether we should connect to the portal on startup. This allows the relay to be used/tested standalone and keeps the number of config options and error cases small.

We require the user to config the full path of the websocket and thus avoid the need for duplicating the connlib function. Given that most users will never need to override this option, this seems like a good trade-off.

Resolves https://github.com/firezone/product/issues/614.

### CODE REVIEW

File: rust/relay/src/main.rs [hunk 1]
```diff
@@ -30,11 +30,11 @@ struct Args {
     #[arg(long, env)]
     listen_ip4_addr: Ipv4Addr,
     /// The websocket URL of the portal server to connect to.
+    #[arg(long, env, default_value = "wss://api.firezone.dev/relay/websocket")]
```
Comment (jamilbk) [inline rust/relay/src/main.rs]: 👍

**Review by jamilbk — APPROVED** [review]
LGTM

**Comment by jamilbk** [comment]
Fixes firezone/product#614

File: rust/relay/src/main.rs (hunk 1 above)
Comment (conectado) [inline rust/relay/src/main.rs]: I'd rather append `relay/websocket` ourselves, although most users won't use that the ones that do will find that less confusing.
The `<component>/websocket` endpoint is a phoenix convention that we didn't pick and we don't know if they will ever change it and break the users' scripts.
However we can expect the user will know the URL of their firezone instance.

Comment (thomaseizinger) [reply rust/relay/src/main.rs]: Done.

**Review by conectado — APPROVED** [review]
LGTM

**Pull request merged**